ospf6d/ospf6_interface.c

```diff
@@ -1745,8 +1746,10 @@ void ospf6_interface_start(struct ospf6_interface *oi)
 	ospf6_interface_enable(oi);
 
 	/* If the router is ABR, originate summary routes */
-	if (ospf6_check_and_set_router_abr(ospf6))
+	if (ospf6_check_and_set_router_abr(ospf6)) {
 		ospf6_abr_enable_area(oa);
+		ospf6_schedule_abr_task(ospf6);
```

Comment:
Are we taking care of stopping the ospf6_abr_task_timer in ospf6_interface_stop ?